Orange/widgets/model/owlogisticregression.py

```diff
@@ -92,10 +92,10 @@ def get_learner_parameters(self):
 def create_coef_table(classifier):
     i = classifier.intercept
     c = classifier.coefficients
-    if len(classifier.domain.class_var.values) > 2:
-        values = classifier.domain.class_var.values
+    if c.shape[0] > 2:
```

Comment:
Isn't this something different? Wasn't the previous code correct? Isn't `classifier.coefficients` equal to the number of attributes + 1? Or am I overlooking something?